pkg/hierarchy.go

```diff
@@ -118,6 +121,7 @@ func GetHierarchy(configFile, applicationName string) (Config, error) {
 		return Config{}, err
 	}
 	for i := range config.Hierarchy {
+		// Exposes a 1-based index API
```

Comment:
Can we add a better description.. Don't understand from current comment

Reply:
@moshloop ?

Reply:
Changed